### PR TITLE
[video][ios] Add live indicator in the now playing info

### PR DESCRIPTION
**Pull request by JustJoostNL**
# Why
If a video was live, it wasn't showing a live indicator in the now playing section, now it will.

# How
I made use of the `MPNowPlayingInfoPropertyIsLiveStream` property, found in the Apple Developer documentation:
https://developer.apple.com/documentation/mediaplayer/mpnowplayinginfopropertyislivestream

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-video/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 ### 🎉 New features
 
+- [iOS] Add live decorator in the now playing info. ([#30629](https://github.com/expo/expo/pull/30629) by [@justjoostnl](https://github.com/justjoostnl))
```
Comment (tsapeta) [inline packages/expo-video/CHANGELOG.md:11]: Was it supposed to be `indicator`? 😅 
```suggestion
- [iOS] Add live indicator in the now playing info. ([#30629](https://github.com/expo/expo/pull/30629) by [@justjoostnl](https://github.com/justjoostnl))
```

**Review by tsapeta — APPROVED** [review]

File: packages/expo-video/CHANGELOG.md (hunk 1 above)
Comment (JustJoostNL) [reply packages/expo-video/CHANGELOG.md]: Yep, thanks! 🙂

**Review by behenate — APPROVED** [review]
Thanks! Have you checked if there was a similar problem on Android?

**Comment by JustJoostNL** [comment]
@behenate I've done some research, but can't seem to find a live property for android, for the now playing notification.

**Pull request merged**